.github/workflows/installer.yml

```diff
@@ -81,6 +81,25 @@ jobs:
       # tar and gzip are needed by the actions/checkout below.
       run: zypper --non-interactive install tar gzip ${{ matrix.deps }}
       if: contains(matrix.image, 'opensuse')
+      # Parrot recently (~2025-01-11) rotated their GPG keys for package signing
+      # but have not (as of 2025-01-24) updated their LTS image to include these
+      # new GPG keys. This image does not come with CURL / wget / a sane way of
+      # fetching the .deb file that their blog post around the key rotation points
+      # to (see https://parrotsec.org/blog/2025-01-11-parrot-gpg-keys/) so instead
+      # use a combination of openssl s_clients and sed to download the .deb and strip
+      # the header information from the resulting file so that it is a a valid .deb
+      # that we can install.
+    - name: install new GPG key (parrotsec/core:lts-amd64)
+      run: |
+        printf "GET /parrot/pool/main/p/parrot-archive-keyring/parrot-archive-keyring_2024.12_all.deb HTTP/1.1\r\nHost: www.deb.parrot.sh\r\nConnection: close\r\n\r\n" | openssl s_client -connect www.deb.parrot.sh:443 -quiet | sed  '/<arch>/,$!d' > parrot-archive-keyring_2024.12_all.deb 
```

Comment:
this is art.

Reply:
Definitely one way to describe this 🤾‍♂️